doc/lua_api.txt

```diff
@@ -2434,8 +2434,12 @@ These functions return the leftover itemstack.
     * Optional: Variable number of arguments that are passed to `func`
 
 ### Server
-* `minetest.request_shutdown([message],[reconnect])`: request for server shutdown. Will display `message` to clients,
-    and `reconnect` == true displays a reconnect button.
+* `minetest.request_shutdown([message],[reconnect],[delay])`: request for server shutdown. Will display `message` to clients,
+    `reconnect` == true displays a reconnect button,
+    `delay` permit to add an optional delay before shutdown
```

Comment:
"permit to add"?
What is the delay measured in?  Seconds?  Milliseconds?  Years?